spec/models/theme_spec.rb

```diff
@@ -341,10 +341,10 @@ def transpile(html)
       expect(scss).to include("font-size:30px")
 
       # Escapes correctly. If not, compiling this would throw an exception
-      setting.value = <<~CSS
+      setting.value = <<~SCSS
```

Comment:
Sneaking in this cosmetic change here because it messes up the syntax highlighting in VScode for the rest of the file.